.github/workflows/build_webrtc.yml

```diff
@@ -0,0 +1,58 @@
+name: Build WebRTC
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch_number:
+        description: 'The branch number of WebRTC to build'
+        required: true
+      milestone:
+        description: 'The milestone number of the WebRTC branch'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4.1.1
+
+    - name: Build Docker image
+      run: |
+        docker build -t webrtc-android .
+
+    - name: Create Docker Volume
+      run: |
+        docker volume create webrtc-src
+
+    - name: Run the build process with Docker
+      uses: addnab/docker-run-action@v3
```

Comment:
out of curiosity, could we substitute this with a script that does a bunch of `docker exec` i.e. 
```
docker exec -w /webrtc-android webrtc-android python3 build.py build
```